src/pages/publication/DescriptionPanel.tsx

```diff
@@ -62,7 +62,7 @@ const DescriptionPanel: FC<DescriptionPanelProps> = ({ goToNextTab, savePublicat
 
   // Validation messages won't show on fields that are not touched
   const setAllFieldsTouched = useCallback(() => {
-    Object.values(DescriptionFieldNames).forEach(fieldName => setFieldTouched(fieldName));
+    Object.values(DescriptionFieldNames).forEach((fieldName) => setFieldTouched(fieldName));
```

Comment:
ahh, dette som er det nye i prettier ja

Reply:
why though? virker unødvendig

Reply:
ryddigere uten, ja
men fra doc: "At first glance, avoiding parentheses may look like a better choice because of less visual noise. However, when Prettier removes parentheses, it becomes harder to add type annotations, extra arguments or default values as well as making other changes. Consistent use of parentheses provides a better developer experience when editing real codebases, which justifies the default value for the option."

Reply:
vi kan jo selvsagt override om vi vil, da

Reply:
forsåvidt logisk mtp typing da. det er jo faktisk noe jeg har irritert meg over når jeg tenker meg om 😆

Reply:
ja, jeg også :P men det blir litt ekstra rot og mas nå når det plutselig håndheves overalt da :(